packages/web-components/src/components/ic-toggle-button-group/test/ic-toggle-button-group.spec.ts

```diff
@@ -0,0 +1,216 @@
+import { newSpecPage } from "@stencil/core/testing";
+import { ToggleButtonGroup } from "../ic-toggle-button-group";
+import { ToggleButton } from "../../ic-toggle-button/ic-toggle-button";
+import { Button } from "../../ic-button/ic-button";
+
+const keyboard = (key: string): KeyboardEvent => {
+  return new KeyboardEvent("keydown", { key: key });
+};
+
+describe("ic-toggle-button-group component unit tests", () => {
+  it("getAllToggleButtons should return an array of slotted ic-toggle-buttons", async () => {
+    const page = await newSpecPage({
+      components: [ToggleButtonGroup, ToggleButton, Button],
+      html: `        <ic-toggle-button-group
+            select-type="single"
+            size="small"
+          >
+            <ic-toggle-button label="Toggle"></ic-toggle-button>
+            <ic-toggle-button label="Toggle"></ic-toggle-button>
+            <ic-toggle-button label="Toggle"></ic-toggle-button>
+          </ic-toggle-button-group>`,
+    });
```

Comment:
There's some weird spacing going on here and in a couple of the other spec tests